samples/net/openthread/coap/src/led.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2024 Alexandre Bailon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef COAP_LED_H
+#define COAP_LED_H
+
+#include <zephyr/data/json.h>
+
+#define LED_URI "led"
+
+#define LED_MSG_STATE_OFF    0
+#define LED_MSG_STATE_ON     1
+#define LED_MSG_STATE_TOGGLE 2
+
+#define JSON_MAX_LED 4
+
+struct json_led_state {
+	int led_id;
+	int state;
+};
+
+struct json_led_get {
+	const char *device_id;
+	struct json_led_state leds[JSON_MAX_LED];
+	int count;
+};
+
+extern const struct json_obj_descr json_led_state_descr[];
+extern const struct json_obj_descr json_led_get_descr[];
```

Comment:
Again, this doesn't seem to be used outside of `led.h`, I'd just remove the declarations here.